### DISCUSSION TITLE
Lossless rotating of videos

### DISCUSSION THREAD

**Discussion by Nedrozak** [mifi/lossless-cut#1283] [Ideas]
Hello! I want to say that this is the best tool on the Internet for manipulating video files. 
I want to request a lossless rotation to be added with the _-metadata:s:v:0 rotate="XX"_. And if it could be added to batch aswell it would be amazing.

**Comment by mifi** [comment]
Hi. Theres already a rotate function. Is it not what you need? Or do you mean to rotate the subtitles?
